pysb/tests/test_util.py

```diff
@@ -0,0 +1,41 @@
+from pysb.core import *
+from pysb.util import *
+from pysb.examples import earm_1_0
+
+
+def test_alias_model_components():
+    """
+    Tests that alias_model_components() exports to the global namespace
+    """
+    m = Model(_export=False)
+    m.add_component(Monomer('temp_monomer_A', _export=False))
+    try:
+        temp_monomer_A
+        raise NameError('Monomer A should not be defined in global namespace')
+    except NameError:
+        # We expect this exception due to _export=False
+        pass
+    alias_model_components(m)
+
+    # Use global version of temp_monomer_A (needed to delete it)
```

Comment:
You can just delete straight out of globals with `del globals()['temp_monomer_A']` so this `global` declaration isn't needed.